literature.bib

```diff
@@ -11242,6 +11615,70 @@ @inproceedings{KTS+:ICSE09
 
 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%% 2008 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{S:SPLC08,
+	title = {{Automated Analysis of Feature Models Using Atomic Sets}},
+	author = {Segura, Sergio},
+	booktitle = SPLC,
+	location = {Limerick, Ireland},
+	pages = {201--207},
+	month = SEP,
+	volume = 2,
+	publisher = IEEE,
+	address = Washington,
+	year = 2008
+}
+
+@inproceedings{NAM:Euromicro08,
+  title = {{Income: Integrated Cost Model for Product Line Engineering}},
+  author = {N{\'o}brega, Jarley Palmeira and de Almeida, Eduardo Santana and Meira, S{\'\i}lvio Romero Lemos},
+  booktitle = {Proc.\ of Euromicro Conf.\ on Software Engineering and Advanced Applications},
+  pages = {27--34},
+  year = 2008,
+  publisher = IEEE,
+  address = Washington,
+  location = {Parma, Italy}
+}
+
+@article{B:JSAT08,
+  title = {{PicoSAT Essentials}},
+  author = {Biere, Armin},
+  journal = JSAT,
+  publisher = IOS,
+  volume = {4},
+  pages = {75--97},
+  year = {2008}
+}
+
+@inproceedings{MB:TACAS08,
+	title = {{Z3: An Efficient SMT solver}},
+	author = {De Moura, Leonardo and Bj{\o}rner, Nikolaj},
+	booktitle = TACAS,
+	pages = {337--340},
+	publisher = Springer,
+	address = BerlinHeidelberg,
+	year = 2008
+}
+
+@inproceedings{JRL:OSSICP08,
+  title = {{Choco: An Open Source Java Constraint Programming Library}},
+  author = {Jussien, Narendra and Rochart, Guillaume and Lorca, Xavier},
+  booktile = {Proc.\ Workshop on Open-Source Software for Integer and Contraint Programming (OSSICP)},
```

Comment:
Put this in MYabrv and MYshort? Because when using MYshort for your paper, this will still be put displayed in this very long form and would be inconsistent in you latex document (w.r.t. to the other references).